src/polyswarm/client/report.py

```diff
@@ -15,31 +15,39 @@ def report():
 
 
 SECTIONS = "summary, detections, fileMetadata, network, droppedFiles, extractedConfig, analysis"
+SANDBOX_ARTIFACT_TYPES = "report, raw_report, screenshot, recording, dropped_file, memory_dump, pcap, jarm"
 
 
 @report.command('create', short_help='Create a report for an instance or sandbox id.')
-@click.argument('format', type=click.Choice(['html', 'pdf']))
-@click.argument('type', type=click.Choice(['scan', 'sandbox']))
+@click.argument('format', type=click.Choice(['html', 'pdf', 'zip']))
+@click.argument('type', type=click.Choice(['scan', 'sandbox', 'sandbox_zip']))
```

Comment:
Do we need a new type? if this is needed to be passed to the API, I think it's an implementation detail that has to be hidden. For end users it's just a sandbox report in the format of zip:

```
polyswarm report create zip sandbox SANDBOX_TASK_ID
```

Reply:
Yes good point, updated in [344e40b](https://github.com/polyswarm/polyswarm-cli/pull/174/commits/344e40b87d6826da910a8e37dcf9d5826572126d)